app/src/main/java/org/xwiki/android/sync/bean/XWikiUserFull.java

```diff
@@ -121,7 +122,8 @@ public void setPhone(String phone) {
      * @return Avatar URL without base url
      */
     public String getAvatar() {
-        return "bin/download/XWiki/" + pageName
+        return ApiEndPoints.BIN + ApiEndPoints.DOWNLOAD
+            + ApiEndPoints.STANDARD_XWIKI_WIKI + pageName
```

Comment:
You are still hardcoding "XWiki", you just stored it in a variable. What I meant is that you should get it from the same place where you got the page name.

Reply:
Fixed